<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lamps</class>
 <widget class="QDialog" name="lamps">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1267</width>
    <height>712</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>185</red>
        <green>115</green>
        <blue>117</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>185</red>
        <green>115</green>
        <blue>117</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>185</red>
        <green>115</green>
        <blue>117</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>185</red>
        <green>115</green>
        <blue>117</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>40</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>45</red>
         <green>45</green>
         <blue>52</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>45</red>
         <green>45</green>
         <blue>52</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled/>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Britannic Bold</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>LAMPS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lamp1btn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>260</y>
     <width>301</width>
     <height>291</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="lamp2btn">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>260</y>
     <width>301</width>
     <height>291</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="lamp3btn">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>260</y>
     <width>301</width>
     <height>291</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
